include/ensmallen_bits/nsga2/nsga2_impl.hpp

```diff
@@ -278,7 +278,7 @@ template<typename MatType>
 inline void NSGA2::Crossover(MatType& childA,
                              MatType& childB,
                              const MatType& parentA,
-                             const MatType& parentB)
+                             const MatType&  parentB)
```

Comment:
```suggestion
                             const MatType& parentB)
```

Let me provide a quick patch for that :+1: